bin/test_send.py

```diff
@@ -1,26 +1,26 @@
-import pika 
+import pika
 import os
 import yaml
 
 rmq_user = os.environ['RMQ_USR']
 rmq_pwd = os.environ['RMQ_PWD']
 
-con = pika.BlockingConnection(pika.URLParameters(f'amqp://{rmq_user}:{rmq_pwd}@141.142.238.15:5672/%2ftest_at'))
+con = pika.BlockingConnection(pika.URLParameters(f'amqp://{rmq_user}:{rmq_pwd}@141.142.238.15:5672/%2ftest_cc'))
 ch = con.channel()
 
-image_id = 'AT_O_20191025_000001'
+image_id = 'gen1'
 
 xfer = {}
 xfer["RAFT_LIST"] = "00"
-xfer["RAFT_CCD_LIST"] = [ "00" ]
+xfer["RAFT_CCD_LIST"] = [ "00/0" ]
 xfer["AT_FWDR"] = "f99"
 
 msg = {}
 msg["MSG_TYPE"] = "AT_FWDR_XFER_PARAMS"
 msg["REPLY_QUEUE"] = "at_foreman_ack_publish"
 msg["IMAGE_ID"] = image_id
 msg["ACK_ID"] = "ack_100"
-msg["TARGET_LOCATION"] = "ARC@141.142.238.15:/data/export"
+msg["TARGET_LOCATION"] = "hwin16@141.142.238.15:/home/hwin16"
```

Comment:
It would probably be better to use ARC for this, instead of hwin16.   If the concern is where the data is being put during testing, put it into a new directory, like /data/daqtest or something like that.

Reply:
got it!